packages/core/index.ts

```diff
@@ -136,7 +136,7 @@ export function createEnv<
       );
     });
 
-  if (parsed.success === false) {
+  if (!parsed.success) {
```

Comment:
Revert. For some reason just the ! operator doesnt work for some. We've gotten issues in create-t3-app with it instead of explicit === false